cmd/dsa_plugin/README.md

```diff
@@ -0,0 +1,130 @@
+# Intel DSA device plugin for Kubernetes
+
+Table of Contents
+
+* [Introduction](#introduction)
+* [Installation](#installation)
+    * [Deploy with pre-built container image](#deploy-with-pre-built-container-image)
+    * [Getting the source code](#getting-the-source-code)
+    * [Verify node kubelet config](#verify-node-kubelet-config)
+    * [Deploying as a DaemonSet](#deploying-as-a-daemonset)
+        * [Build the plugin image](#build-the-plugin-image)
+        * [Deploy plugin DaemonSet](#deploy-plugin-daemonset)
+    * [Deploy by hand](#deploy-by-hand)
+        * [Build the plugin](#build-the-plugin)
+        * [Run the plugin as administrator](#run-the-plugin-as-administrator)
+    * [Verify plugin registration](#verify-plugin-registration)
+
+## Introduction
+
+The DSA device plugin for Kubernetes supports acceleration using the Intel Data Streaming accelerator(DSA).
+
+The DSA plugin discovers DSA work queues and presents them as a node resources.
+
+## Installation
+
+The following sections detail how to obtain, build, deploy and test the DSA device plugin.
+
+Examples are provided showing how to deploy the plugin either using a DaemonSet or by hand on a per-node basis.
+
+### Deploy with pre-built container image
+
+[Pre-built images](https://hub.docker.com/r/intel/intel-dsa-plugin)
+of this component are available on the Docker hub. These images are automatically built and uploaded
+to the hub from the latest master branch of this repository.
+
+Release tagged images of the components are also available on the Docker hub, tagged with their
+release version numbers in the format `x.y.z`, corresponding to the branches and releases in this
+repository. Thus the easiest way to deploy the plugin in your cluster is to run this command
+
+```bash
+$ kubectl apply -k https://github.com/intel/intel-device-plugins-for-kubernetes/deployments/dsa_plugin?ref=<RELEASE_VERSION>
+daemonset.apps/intel-dsa-plugin created
+```
+
+Where `<RELEASE_VERSION>` needs to be substituted with the desired release version, e.g. `v0.19.0`.
```

Comment:
nit: there's no `v0.19.0` for DSA

Reply:
Well, there is no any release for DSA, it's just an example.

Reply:
true, it was just a small nit that we could make the example something that really exists (some day) to avoid any confusion.

Reply:
ok, I'll change it to point to the existing ref. thanks.

Reply:
done